src/PKSim.Core/Model/Formulation.cs

```diff
@@ -40,6 +41,26 @@ public Formulation() : base(PKSimBuildingBlockType.Formulation)
       ///    returns true if the formulation is defining a dissolved formulation
       /// </summary>
       public bool IsDissolved => string.Equals(FormulationType, CoreConstants.Formulation.Dissolved);
+      
+      /// <summary>
+      /// Returns <c>true</c> if the formulation is particle formulation using monodisperse system otherwise <c>false</c>
```

Comment:
I moved all of this logic out of the presenter and in the formulation object. Why not use it since we already have it